cirq/contrib/quantum_volume/quantum_volume.py

```diff
@@ -165,21 +167,30 @@ def compile_circuit(
     # target qubits given our topology.
     if router is None and routing_algo_name is None:
         routing_algo_name = 'greedy'
-    swap_network = ccr.route_circuit(compiled_circuit,
-                                     ccr.xmon_device_to_graph(device),
-                                     router=router,
-                                     algo_name=routing_algo_name)
-    compiled_circuit = swap_network.circuit
+
+    best_swap_network: Union[ccr.SwapNetwork, None] = None
+    best_score = -1
```

Comment:
extreme nit: this seems like a weird initial value. I typically use `None` for these cases.

Reply:
Done.